src/UI Component/Adminpanel/Activities/postloanrepayment/posttransactionform.js

```diff
@@ -0,0 +1,15 @@
+// import AdminLoanRepaymentPage from "./posttransact";
+import LoanRepaymentForm from "./posttransaction";
```

Comment:
Naming of this should be `post-transaction` or something better as explained above